sdk/monitor/azure-monitor-query/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.4 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
This would be a minor version bump only if new features are added. Keep in mind when you are actually doing the release